runtime_builders/publish_builders.py

```diff
@@ -63,9 +63,8 @@ def _publish_latest(builder_dir):
         project_name = config['project']
 
         parts = os.path.splitext(latest)
-        project_prefix = os.path.join(builder_util.RUNTIME_BUCKET_PREFIX,
-                                      project_name, '-')
-        latest_file = parts[0][len(project_prefix):]
+        prefix = builder_util.RUNTIME_BUCKET_PREFIX + project_name + '-'
```

Comment:
the existing prefix is `gs://runtime-builders/`, so this makes it `gs://runtime-builders/<project_name>-`.